cluster/gce/util.sh

```diff
@@ -978,21 +978,37 @@ function test-setup {
 
   # Open up port 80 & 8080 so common containers on minions can be reached
   # TODO(roberthbailey): Remove this once we are no longer relying on hostPorts.
+  local start=`date +%s`
```

Comment:
I increased timeout to 20 minutes, but kept start here, to avoid depending on 11 min gcloud timeout.